pxtblocks/blocklyloader.ts

```diff
@@ -11,7 +11,9 @@ namespace pxt.blocks {
         images: 45,
         variables: 330,
         text: 160,
-        lists: 260
+        lists: 260,
+	math: 230,
```

Comment:
If you run [tslint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) in VS code, this indentation will be fixed automatically by pressing Shift+Alt+F